.github/scripts/commit_and_push.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+readonly GIT_HAS_CHANGES=1 # 0 = clean, 1 = dirty
+
+USER_NAME="$1"
+USER_EMAIL="$2"
+
+git config --global user.name "$USER_NAME"
+git config --global user.email "$USER_EMAIL"
+
+git diff --quiet; GIT_DIFF_STATUS=$?
```

Comment:
Nice! Can we add a small comment that explains this?

Reply:
Done